docs/tutorial/part-zero/index.md

```diff
@@ -55,13 +57,21 @@ npm has three distinct components: the npm website, the npm registry, and the np
 
 > 💡 Check out npm’s introduction, “[**What is npm?**](https://docs.npmjs.com/getting-started/what-is-npm)”.
 
-## Install Git
+## Git
+
+Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. When you install a Gatsby "starter" site, Gatsby uses Git behind the scenes to download and install the required files for your starter. You will need to have Git installed to set up your first Gatsby site.
+
+### ✋ Install Git
+
+The steps to download and install Git depend on your operating system. Follow the guide for your system:
 
-Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. When you install a Gatsby "starter" site, Gatsby uses Git behind the scenes to download and install the required files for your starter.
+- [Install Git on macOS](https://www.atlassian.com/git/tutorials/install-git#mac-os-x)
+- [Install Git on Windows](https://www.atlassian.com/git/tutorials/install-git#windows)
+- [Install Git on Linux](https://www.atlassian.com/git/tutorials/install-git#linux)
 
-If your system does not have Git installed, install it from the [**Git downloads page**](https://git-scm.com/downloads).
+> 💡 You will not need to use Git yourself to work with Gatsby, but it is a very useful tool to know about as a developer. If you are interested in learning more about version control, Git, and Github, check out Github's [Git Handbook](https://guides.github.com/introduction/git-handbook/).
 
-## Install the Gatsby CLI
+## The Gatsby CLI
```

Comment:
The "the" here is unnecessary. What I mean is, it adds no meaning. The Gatsby CLI would make sense in a world where this is the __only__ Gatsby CLI ever. 😆

Reply:
I agree!

Reply:
You are right I have changed this title and other references to it to make it more clear that it is not the only Gatsby CLI